src/NzbDrone.Core/Notifications/Simplepush/SimplepushSettings.cs

```diff
@@ -0,0 +1,34 @@
+using System.Collections.Generic;
+using FluentValidation;
+using NzbDrone.Core.Annotations;
+using NzbDrone.Core.ThingiProvider;
+using NzbDrone.Core.Validation;
+
+namespace NzbDrone.Core.Notifications.Simplepush
+{
+    public class SimplepushSettingsValidator : AbstractValidator<SimplepushSettings>
+    {
+        public SimplepushSettingsValidator()
+        {
+            RuleFor(c => c.Key).NotEmpty();
+        }
+    }
+
+    public class SimplepushSettings : IProviderConfig
+    {
+        private static readonly SimplepushSettingsValidator Validator = new SimplepushSettingsValidator();
+
+        [FieldDefinition(0, Label = "Key", Privacy = PrivacyLevel.ApiKey, HelpLink = "https://simplepush.io/features")]
+        public string Key { get; set; }
+
+        [FieldDefinition(1, Label = "Event", HelpLink = "https://simplepush.io/features")]
+        public string Event { get; set; }
```

Comment:
What does Event do here? With it being an open string I assume the value can be anything? Some help text might be nice for users.

Reply:
Events can be defined in the Simplepush app. They are used to customize the behavior (sound, vibration pattern, critical alert, ...) of push notification received by the app. I just added the help text. Thank you for pointing this out.

Reply:
Is there an API request to retrieve user defined events so this can be presented in a select drop-down or no?

Reply:
No, there is no way of finding out if a user defined the event in the app or not. If the user didn't define the event but sends a notification with an event, the notification is shown with the default behavior.